src/sentry/models/group.py

```diff
@@ -448,6 +464,8 @@ class Meta:
             ("project", "id"),
             ("project", "status", "last_seen", "id"),
             ("project", "status", "type", "last_seen", "id"),
```

Comment:
Keeping these old indexes around since they'll still be used.

Reply:
I believe there are some custom partial indexes on this table (only in Saas) as well. We probably don't need to reproduce them yet with substatus, but something to be aware of.

Reply:
Which partial indexes are you referring to? The ones OPS had to create for the backfill?